app/models/report.rb

```diff
@@ -51,6 +51,14 @@ def destroy_reported_item
     mark_as_reviewed
   end
 
+  def mark_as_nsfw
```

Comment:
>  Give me a federated-solution and I change it right away

Mark the profile as NSFW or delete it. I'm not going to support adding inconsistencies by locally modifying posts, sorry. Maybe others have different opinions, though.